src/main/resources/screens/live/IEDET_SEND_INPUT.json

```diff
@@ -22,18 +24,40 @@
       },
       "component": "entity-list",
       "name": "FinalResponseDocument",
-      "label": "Upload the Final Response"
+      "label": "Upload the final response"
     },
     {
       "validation": [
         "required"
       ],
       "props": {
-        "choices": "S_IEDET_CASE_OUTCOME"
+        "choices": [
+          {
+            "label": "Substantiated",
+            "value": "substantiated"
+          },
+          {
+            "label": "Unsubstantiated",
+            "value": "unsubstantiated"
+          },
+          {
+            "label": "Partially Substantiated",
+            "value": "partiallysubstantiated"
```

Comment:
Would having the values in pascal case be a bit more consistent with the dropdowns in other places?